client/app/lib/base_view.es6

```diff
@@ -1,12 +1,12 @@
-module.exports = class BaseView extends Backbone.View {
-  constructor() {
-    super();
-  }
 
-  template() {}
+export default class BaseView extends Backbone.View {
+
+  template() {
+    return '';
+  }
 
   getRenderData() {
-    return { model: this.model.toJSON() };
+    return { model: JSON.stringify(this.model) };
```

Comment:
Oups, that's right... After consieration it seem not be the best idea to automatically send the object data so I will follow the JS template and return null by default.